clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -750,6 +750,11 @@ private void createServerStore(ClientDescriptor clientDescriptor, CreateServerSt
 
     serverStore.setEvictionListener(key -> invalidateHashAfterEviction(name, key));
     attachStore(clientDescriptor, name);
+    try {
+      entityMessenger.messageSelfAndDeferRetirement(createServerStore, new ClientIDTrackerMessage.ServerStoreLifeCycleReplicationMessage(createServerStore));
+    } catch (MessageCodecException e) {
+      LOGGER.error("Codec Exception", e);
```

Comment:
Let's throw `AssertionError` as this should only be a programming error, nothing else. Same for the similar block below.